node/libs/crypto/src/ed25519/mod.rs

```diff
@@ -3,15 +3,17 @@
 
 use crate::ByteFmt;
 use anyhow::Context as _;
-use ed::{Signer as _, Verifier as _};
+use ed::Signer as _;
 use ed25519_dalek as ed;
+use zeroize::ZeroizeOnDrop;
 
 pub mod testonly;
 
 #[cfg(test)]
 mod tests;
 
 /// ed25519 secret key.
+#[derive(ZeroizeOnDrop)]
```

Comment:
It is [enabled](https://github.com/dalek-cryptography/ed25519-dalek?tab=readme-ov-file#feature-flags) by default, but let's keep it here